flutter/realm_flutter/android/build.gradle

```diff
@@ -80,10 +85,17 @@ task prePreBuild(type: Exec) {
         commandLine dartPath, 'run', 'realm', 'metrics', '--verbose', '--flutter-root', flutterRoot, '--target-os-type', 'android', '--target-os-version', targetOsVersion
```

Comment:
```suggestion
        commandLine dartPath, 'run', 'realm', 'metrics', '--flutter-root', flutterRoot, '--target-os-type', 'android', '--target-os-version', targetOsVersion
```

Reply:
This does not show if you do `flutter run` or `flutter build`. It only shows with `flutter run|debug --verbose`. I wonder if we should leave it on. This will output the json data right?

Reply:
Yes

Reply:
fixed. removed the --verbose flag